cli/cmd/utils_internal_test.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2020 The Lokomotive Authors
```

Comment:
I'd like a test where both the KUBECONFIG flag is set and the lokocfg file is present to make sure it takes the file defined in the lokocfg. This is the most dangerous case IMO.